content/kotsadm/installing/installing-embedded-cluster.md

```diff
@@ -6,11 +6,11 @@ weight: 10040
 draft: false
 ---
 
-This article refers to installing the Admin Console on an embedded cluster. 
+This article refers to installing the Admin Console on an embedded cluster.
 When running the Admin Console on an existing cluster, refer to the [Installing the Admin Console](/kotsadm/installing/installing-a-kots-app/) documentation.
 
 ### Powered by kURL
-Replicated KOTS leverages a [deep integration](https://blog.replicated.com/kurl-with-replicated-kots/) with the [Replicated kURL project](https://github.com/replicatedhq/kurl) in order to provide native embedded Kubernetes cluster support. 
+Replicated KOTS leverages a [deep integration](https://blog.replicated.com/kurl-with-replicated-kots/) with the [Replicated kURL project](https://github.com/replicatedhq/kurl) in order to provide native embedded Kubernetes cluster support.
 More documentation on installing with kURL (including [advanced install options](https://kurl.sh/docs/install-with-kurl/advanced-options)) is available at [kurl.sh/docs](https://kurl.sh/docs).
 
 ### Online Installations
```

Comment:
So, the airgap instructions explain installing both admin console and the app, yet this section only covers admin console and gives no instruction for how to actually finish installing your app. Where are those instructions?